### PR TITLE
Mobile support

### PR DESCRIPTION
**Pull request by MondayHopscotch**

### CODE REVIEW

File: Project.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	<window width="450" height="800" fps="60" background="#000000" hardware="true" vsync="false" />
 
 	<!--HTML5-specific-->
-	<window if="html5" resizable="false" />
+	<window if="html5" resizable="true" />
```
Comment (MondayHopscotch) [inline Project.xml:20]: @jacque006 @thavelick This was why it didn't render correctly on web fullscreen....

Comment (jacque006) [reply Project.xml:20]: Do we want to follow up by adding those swipe/tap controls? Or does it already work?

Comment (thavelick) [reply Project.xml:20]: Heh, that's easy. Swipe controls do indeed work as far as I could tell.

Comment (jacque006) [reply Project.xml:20]: Noice!

Comment (MondayHopscotch) [reply Project.xml:20]: I put swipe controls in this morning, heh. It was super simple (might not work for other games... but worked for this one)

**Pull request merged**